atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -45,6 +45,9 @@ private AtlasDbConstants() {
     public static final TableReference DEFAULT_SCHEMA_METADATA_TABLE = TableReference.createWithEmptyNamespace(
             "_schema_metadata");
 
+    // Deprecated tables
+    public static final TableReference SWEEP_PROGRESS_V2 = TableReference.createWithEmptyNamespace("_sweep_progress2");
```

Comment:
There used to be `sweep_progress` (the version 1) too, but I think that was sweepable, so dont need to blacklist it :).